src/main/flight/pid.c

```diff
@@ -614,6 +627,29 @@ void pidAcroTrainerInit(void)
 }
 #endif // USE_ACRO_TRAINER
 
+float pidCompensateThrustLinearization(float throttle)
+{
+#ifdef USE_THRUST_LINEARIZATION
```

Comment:
These can go outside of the function body.

Reply:
That was done on purpose to limit the degree of conditionalization. Conditionals in the body allows the caller not to have conditionals which don't add value since they don't reduce code size. Acceptable?

Reply:
See my remark above.